concourse/steps/component_descriptor.mako

```diff
@@ -76,15 +77,28 @@ component_v2 = base_descriptor_v2.component
 
 ## XXX unify w/ injection-method used for main-repository
 % for repository in other_repos:
+<%
+  repo_variable_name = f"repo_{other_repos.index(repository)}"
```

Comment:
this is a very hard-to-read codestyle. please do not partially construct variable names like that